lib/mapper.js

```diff
@@ -113,7 +113,7 @@ function formatConversionTag(msg) {
   var ret = {
     rdid: device.advertisingId,
     idtype: getIdType(device),
-    lat: device.adTrackingEnabled ? 1 : 0,
+    lat: device.adTrackingEnabled ? 0 : 1,
```

Comment:
According to https://developers.google.com/app-conversion-tracking/ios/remarketing-server

```
You must indicate whether the device has "limit ad tracking" enabled. If so, set lat=1. If limit ad tracking is not enabled, set lat=0.
```

Unfortunately it's not more specific than that but they refer to this:
https://support.apple.com/en-us/HT202074

It's clear to me that if the user limits adverts in his iOS settings this flag should be set to `1` which is the opposite what you are currently doing.